invenio_app_rdm/config.py

```diff
@@ -635,6 +635,14 @@ def files_rest_permission_factory(obj, action):
         "content-type": "application/x-dc+xml",
         "filename": "{id}.xml",
     },
+    "marcxml": {
+        "name": _("MARCXML"),
+        "serializer": (
+            "invenio_rdm_records.resources.serializers:" "MARCXMLSerializer"
+        ),
+        "content-type": "application/marcxml+xml",
```

Comment:
note to self: https://www.iana.org/assignments/media-types/application/marcxml+xml